step-templates/azure-web-app-start.json

```diff
@@ -0,0 +1,35 @@
+{
+  "Id": "96ce9c63-91b5-4773-9511-830a2e106083",
+  "Name": "Azure Web App - Start",
+  "Description": "Starts an azure web app.\n<hr />\n\n*<p>Note This template is designed to run against an azure web app octopus target </p>*\n*<p>Depends on Azure CLI and powershell to be installed on the running machine</p>*",
+  "ActionType": "Octopus.AzurePowerShell",
+  "Version": 1,
+  "CommunityActionTemplateId": null,
+  "Packages": [],
+  "Properties": {
+    "Octopus.Action.Script.ScriptSource": "Inline",
+    "Octopus.Action.Script.Syntax": "PowerShell",
+    "OctopusUseBundledTooling": "True",
```

Comment:
Could we switch `OctopusUseBundledTooling` to 'False'?  
Same for the step below.

Reply:
updated in commit #1164790